Public/Src/Cache/ContentStore/Distributed/NuCache/FlushableCache.cs

```diff
@@ -157,8 +177,15 @@ private void PerformFlush(OperationContext context)
                     }
                 }
 
+                statistics.CleanupTime = stopwatch.Elapsed;
+                statistics.Leftover = _flushingCache.Count;
+
                 _database.Counters[ContentLocationDatabaseCounters.TotalNumberOfCompletedCacheFlushes].Increment();
             }
+
+            statistics.Growth = _cache.Count;

```

Comment:
Should it be a diff between the old count and the new count?

Reply:
No, the old cache is cleared as soon as we perform the exchange, and "Growth" is meant to measure "how many entries have been added since the flush started".

This gives us an idea on how much we load we had between flushes, and allows us to infer if we are doing many back-to-back flushes.